src/components/FeatureCard/FeatureCard.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { ResponsiveImage, TextGroup } from '..';
+import styles from './FeatureCard.scss';
+
+const responsiveImagePropsMapping = {
+	small: {
+		fallbackWidth: 220,
+		fallbackHeight: 122,
+		sizes: '220px',
+		widthRange: [ 220, 220 ],
+	},
+	large: {
+		fallbackWidth: 220,
+		fallbackHeight: 122,
+		sizes: `
+			(min-width: 1200px) 666px,
+			(min-width: 768px) 346px,
+			220px,
+		`,
+		widthRange: [ 220, 666 ],
+	},
+	smallPortrait: {
+		fallbackWidth: 285,
+		fallbackHeight: 400,
+		sizes: '285px',
+		widthRange: [ 285, 285 ],
+	},
+	largePortrait: {
+		fallbackWidth: 285,
+		fallbackHeight: 400,
+		sizes: `
+			(min-width: 1200px) 382px,
+			285px,
+		`,
+		widthRange: [ 285, 382 ],
+	},
+};
+
+export default function FeatureCard( props: {
+	/**
+	 * Optional smaller text below the component title. As an example, if the component title
+	 * is "The rise of employee activists," a relevant description might be "Employee
+	 * activists are transforming the workplace—and making companies better in the process."
+	 */
+	description?: string,
+	/**
+	 * Whether the card is for article content; passed through to TextGroup to
+	 * determine color and formatting. Defaults to `true`.
+	 */
+	isArticle?: boolean,
+	/**
+	 * Determines the aspect ratio of the image, e.g. whether it is sized for images that are taller
+	 * than they are wide.
+	 */
+	isPortrait?: boolean,
+	/**
+	 * A short phrase that accompanies the Hed. See [Kicker](/?path=/docs/kicker--default-story).
+	 */
+	kicker?: string,
+	/**
+	 * URL of an image in the WordPress media library. Must support
+	 * resizing using the width (`w`), height (`h`) and crop (`crop`)
+	 * query parameters.
+	 */
+	thumbnailUrl: string,
+	/**
+	 * Shows a typical "play" icon overlaid onto the component image. Use when the card represents
+	 * video content.
+	 */
+	showPlayIcon?: boolean,
+	/**
+	 * Determines the size of the title text and the expected size & dimensions of the image to be
+	 * rendered. The 'small' size will optimize the image for smaller contexts, such as content lists
+	 * at mobile sizes. 'Large' and 'portrait' sizes will optimize for contexts such as Guide cards
+	 * and content lists at desktop sizes. See [ResponsiveImage](/?path=/docs/responsiveimage--default-story).
+	 */
+	size: 'small' | 'medium' | 'large',
+	/**
+	 * Primary text, used in TextGroup as Hed. See [Hed](/?path=/docs/hed--default-story).
+	 */
+	title: string,
+} ) {
+	const {
+		isArticle = true,
+	} = props;
+	const imageSize = 'medium' === props.size ? 'large' : props.size; // default to large resolution for medium text
+	const imagePropsSize = props.isPortrait ? `${imageSize}Portrait` : imageSize;
+	return (
+		<div>
+			<div className={`${styles.imageContainer} ${styles[ imageSize ]}`}>
```

Comment:
unused className

Reply:
it would probs be good to use it to set a max-width based on the maximum image width